FreeITCourse/src/lesson03/homework1.java

```diff
@@ -0,0 +1,22 @@
+package lesson03;
+// 1)Начав тренировки, спортсмен в первый день пробежал 10 км.
+// Каждыйдень он увеличивал дневную норму на 10% нормы предыдущего дня.
+// Какой суммарный путь пробежит спортсмен за 7 дней?
+
+import java.text.DecimalFormat;
+
+public class homework1 {
```

Comment:
Class names, methods and variables should use CamelCase:
Class/Interface name starts from capital laters
method/variable from lowercase letter